delta/delta_1.5.x_dss_rename_xxx_fk_overflow_characteristic_to_fk_overflow_char.sql

```diff
@@ -0,0 +1,133 @@
+-- drop connected views
+DROP VIEW IF EXISTS qgep_od.vw_leapingweir;
```

Comment:
@ponceta Does it need these DROP VIEW statements with the current architecture?

Reply:
No it is not : The drop views script runs before pum upgrade : https://github.com/QGEP/datamodel/blob/master/view/drop_views.sql

Reply:
OK thanks - I adjusted the delta